bip-0038.mediawiki

```diff
@@ -4,7 +4,7 @@
   Title: Passphrase-protected private key
   Author: Mike Caldwell <mcaldwell@swipeclock.com>
           Aaron Voisine <voisine@gmail.com>
-  Comments-Summary: No comments yet.
```

Comment:
Comments-Summary: Unanimously Discourage for implementation
 
 
Leave a comment
ไม่ได้เลือกไฟล์ใด
Attach files by dragging & dropping, selecting or pasting them.